i18n/en.toml

```diff
@@ -334,8 +343,15 @@ other = "Deployment"
 [learn_more]
 other = "Learn more"
 
+[read_more]
+other = "Read more"
+
+[connect_with_us]
+other = "Connect with us"
+
+
 [traffic_management]
-other = "Traffic Management"
+other = "Reliability"
```

Comment:
is this correct? Traffic Management replaced with Reliability?

Reply:
Yes, this is only used as a feature heading for the front page, and we've changed the heading to "reliability".  I didn't change the key in the toml because we will probably edit this page further.